pyxem/tests/signals/test_differential_phase_contrast.py

```diff
@@ -469,3 +469,83 @@ def test_retain_axes_manager(self):
         assert s_load.axes_manager[2].units == "b"
         assert s_load.axes_manager[1].name == "e"
         assert s_load.axes_manager[2].name == "f"
+
+class TestPhaseRetrieval:
+
+    def setup_method(self):
+        # construct the surface, two point with Gaussian distribution
+        rng = np.linspace(-20,10,num=512)
+        x, y = np.meshgrid(rng, rng)
+        surface = np.exp(-(x**2+y**2)/2) + np.exp(-((x-2)**2+(y+4)**2)/2)
+
+        # x and y phase gradient of the Gaussians, analytical form
+        dx = x*(-np.exp(-x**2/2 - y**2/2)) +\
+            (x-2)*-np.exp((-0.5*(x-2)**2-0.5*(y+4)**2))
+        dy = y*(-np.exp(-x**2/2 - y**2/2)) +\
+            (y+4)*-np.exp((-0.5*(x-2)**2-0.5*(y+4)**2))
+
+        data = np.empty((2, 512, 512))
+        data[0] = dx
+        data[1] = dy
+        self.s = DPCSignal2D(data)
+        self.s.axes_manager.signal_axes[0].axis = rng
+        self.s.axes_manager.signal_axes[1].axis = rng
+
+        # normalise for comparison later
```

Comment:
is this really a "normalise"?

Reply:
Yes I think I was testing other things before... now rescaling them

Reply:
wonder if this is needlessly picky of me